enterprise/frontend/src/embedding-sdk/side-effect-imports.ts

```diff
@@ -0,0 +1,13 @@
+import "@mantine/core/styles.css";
+import "@mantine/dates/styles.css";
+
+import "metabase/css/vendor.css";
+import "metabase/css/index.module.css";
+
+import "metabase/lib/dayjs";
+
+// Import the EE plugins required by the embedding sdk.
+import "sdk-ee-plugins";
+
+// Imports which are only applicable to the embedding sdk, and not the new iframe embedding.
+import "sdk-specific-imports";
```

Comment:
I think these should not live in the new iframe embedding, but we import the side-effect-imports into the new iframe embedding below. Let's move this to SDK only, I think.